### PR TITLE
Test on 5.2.0 on Windows

### PR DESCRIPTION
**Pull request by polytypic**

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
looks like a bug in [ocaml/setup-ocaml](https://github.com/ocaml/setup-ocaml):
```
  D:\a\_actions\ocaml\setup-ocaml\v2\dist\index.cjs:19591
          if (!arg.includes(" ") && !arg.includes("	") && !arg.includes('"')) {
                   ^
  
  TypeError: arg.includes is not a function
      at ToolRunner._uvQuoteCmdArg (D:\a\_actions\ocaml\setup-ocaml\v2\dist\index.cjs:19591:18)
      at ToolRunner._windowsQuoteCmdArg (D:\a\_actions\ocaml\setup-ocaml\v2\dist\index.cjs:19532:23)
      at ToolRunner._getCommandString (D:\a\_actions\ocaml\setup-ocaml\v2\dist\index.cjs:19474:31)
      at ToolRunner.<anonymous> (D:\a\_actions\ocaml\setup-ocaml\v2\dist\index.cjs:19662:51)
      at Generator.next (<anonymous>)
      at D:\a\_actions\ocaml\setup-ocaml\v2\dist\index.cjs:19428:71
      at new Promise (<anonymous>)
      at __awaiter (D:\a\_actions\ocaml\setup-ocaml\v2\dist\index.cjs:19410:14)
      at D:\a\_actions\ocaml\setup-ocaml\v2\dist\index.cjs:19654:52
      at new Promise (<anonymous>)
  
  Node.js v20.8.1
```

**Comment by polytypic** [comment]
> looks like a bug in [ocaml/setup-ocaml](https://github.com/ocaml/setup-ocaml):
> 
> ```
>   D:\a\_actions\ocaml\setup-ocaml\v2\dist\index.cjs:19591
>           if (!arg.includes(" ") && !arg.includes("	") && !arg.includes('"')) {
>                    ^
>   
>   TypeError: arg.includes is not a function
[...]
> Could you open an issue upstream?

Yes, you mean to [setup-ocaml](https://github.com/ocaml/setup-ocaml/issues), although I have no idea what is the cause of the problem here.  I'll open an issue there with a link to here.

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -28,9 +29,8 @@ jobs:
         with:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
           opam-repositories: |
-            dra27: https://github.com/dra27/opam-repository.git#windows-5.0
-            default: https://github.com/ocaml-opam/opam-repository-mingw.git#sunset
-            standard: https://github.com/ocaml/opam-repository.git
+            default: https://github.com/dra27/opam-repository.git#windows-initial
+            allow-prerelease-opam: true
```
Comment (kit-ty-kate) [inline .github/workflows/workflow.yml:33]:
```suggestion
          allow-prerelease-opam: true
```

Comment (kit-ty-kate) [reply .github/workflows/workflow.yml:33]: my bad :(

**Pull request merged**